### PR TITLE
Implement Games data service

### PR DESCRIPTION
**Pull request by Blank13**

### CODE REVIEW

File: Euroleague-Graduate/Euroleague-Graduate/RealmDBManager.swift [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    
+    private let realm: Realm
+    
+    private var count = 0
+    
+    private init() {
+        realm = try! Realm()
+    }
+    
+    func getDataFromRealm() -> Results<GameData> {
```
Comment (ibrahimovich22) [inline Euroleague-Graduate/Euroleague-Graduate/RealmDBManager.swift:24]: rename to getGames

File: Euroleague-Graduate/Euroleague-Graduate/TurkishAirLinesHelper.swift [hunk 2]
```diff
@@ -15,49 +17,171 @@ class TurkishAirLinesHelper {
...
+    
     let urls: Dictionary<GameDataStatus, String> = [
         .schedule : "http://www.euroleague.net/euroleague/api/schedules?seasoncode=E2016",
         .results : "http://www.euroleague.net/euroleague/api/results?seasoncode=E2016"
     ]
     
-    func getGamesTable() {
+    //Should have no parameters and return table of dataBase
+    func getGamesTable(round: String) {
+        games.removeAll()
```
Comment (ibrahimovich22) [inline Euroleague-Graduate/Euroleague-Graduate/TurkishAirLinesHelper.swift:39]: There should be a data service layer to handle the parsing

ApiClient -> GamesApiClient -> GamesDataService

**Review by ibrahimovich22 — CHANGES_REQUESTED** [review]
Remove un-needed prints and commented code

File: Euroleague-Graduate/Euroleague-Graduate/TurkishAirLinesHelper.swift [hunk 3]
```diff
@@ -8,35 +8,60 @@
...
     func getGamesTable(round: String) {
-        self.round = round
-        if games.count == 0 {
-            getSchedule()
+        games.removeAll()
+        let table = RealmDBManager.sharedInstance.getDataFromRealm()
+        print("Data is here")
+        print("table count: \(table.count)")
+        for game in table {
+            let gameData = GameData()
```
Comment (dalia2011) [inline Euroleague-Graduate/Euroleague-Graduate/TurkishAirLinesHelper.swift:44]: I think it is better to create copy method in GameData class that copy object to another one

File: Euroleague-Graduate/Euroleague-Graduate/TurkishAirLinesHelper.swift [hunk 4]
```diff
@@ -100,6 +146,7 @@ class TurkishAirLinesHelper {
                 game.homeTv = try elem["hometv"].value()
```
Comment (dalia2011) [inline Euroleague-Graduate/Euroleague-Graduate/TurkishAirLinesHelper.swift]: what about create parse method in the class of object

**Review by dalia2011 — CHANGES_REQUESTED** [review]

File: Euroleague-Graduate/Euroleague-Graduate/RealmDBManager.swift (hunk 1 above)
Comment (Blank13) [reply Euroleague-Graduate/Euroleague-Graduate/RealmDBManager.swift:24]: done in ImplementUnitTest branch

**Pull request closed without merging**